pkg/outputs/outputs.go

```diff
@@ -56,4 +56,69 @@ var (
 			Symbol: "Hz",
 		},
 	}
+
```

Comment:
These may change. Synse can now give different types between scan and read, which is odd at best.